packages/main/src/TimePicker.hbs

```diff
@@ -11,12 +14,17 @@
 			@click="{{_handleInputClick}}"
 			@ui5-change="{{_handleInputChange}}"
 			@ui5-input="{{_handleInputLiveChange}}"
+			@focusin="{{_onfocusin}}"
+			@input="{{_oninput}}"
 			class="ui5-time-picker-input"
 			@keydown="{{_onkeydown}}"
 	>
 
 	{{#if valueStateMessage.length}}
-		<slot name="valueStateMessage" slot="valueStateMessage"></slot>
+		<slot
+			name="valueStateMessage"
+			slot="valueStateMessage"
```

Comment:
slot attribute seems to be redundant. correct way to use named slot is:

<slot name="slotName"></slot>

Reply:
Done.